app/config/collections/platform.php

```diff
@@ -330,6 +330,72 @@
                 'default' => null,
                 'array' => false,
                 'filters' => ['datetime'],
+            ],
+            [
+                '$id' => ID::custom('authSubscriptionsEnabled'),
```

Comment:
Same issues with vendor locking and stripe, Imagine tomorrow a new product called UltraPayments comes in to the picture and is better than Stripe in every aspect, we need to be able to switch or support both,